<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>441</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Bahnschrift</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>589</width>
      <height>652</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="VLayoutForm">
       <item>
        <layout class="QFormLayout" name="formLayout">
         <property name="verticalSpacing">
          <number>15</number>
         </property>
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="lblTitle_4">
           <property name="font">
            <font>
             <family>Bahnschrift</family>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string>SIGNALS</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="0">
          <widget class="QLineEdit" name="txtName">
           <property name="font">
            <font>
             <family>Bahnschrift</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="placeholderText">
            <string>First Name</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLineEdit" name="txtLastName">
           <property name="font">
            <font>
             <family>Bahnschrift</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="placeholderText">
            <string>Last Name</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QRadioButton" name="rbFemale">
           <property name="font">
            <font>
             <family>Bahnschrift</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Female</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup</string>
           </attribute>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QRadioButton" name="rbMale">
           <property name="font">
            <font>
             <family>Bahnschrift</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Male</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup</string>
           </attribute>
          </widget>
         </item>
         <item row="7" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QComboBox" name="cbDay">
             <property name="font">
              <font>
               <family>Bahnschrift</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="currentText">
              <string/>
             </property>
             <property name="maxVisibleItems">
              <number>31</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cbMouth">
             <property name="font">
              <font>
               <family>Bahnschrift</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="currentText">
              <string/>
             </property>
             <property name="maxVisibleItems">
              <number>31</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cbYear">
             <property name="font">
              <font>
               <family>Bahnschrift</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="currentText">
              <string/>
             </property>
             <property name="currentIndex">
              <number>-1</number>
             </property>
             <property name="maxVisibleItems">
              <number>100</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="lblTitle_5">
           <property name="font">
            <font>
             <family>Bahnschrift</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Birthday:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="VLayoutConfirm" stretch="0">
       <property name="spacing">
        <number>16</number>
       </property>
       <item>
        <layout class="QFormLayout" name="formLayout_2">
         <property name="verticalSpacing">
          <number>15</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="lblTitle">
           <property name="font">
            <font>
             <family>Bahnschrift</family>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="text">
            <string>SLOTS</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="lblTitle_2">
           <property name="font">
            <font>
             <family>Bahnschrift</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Name:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="lblName">
           <property name="font">
            <font>
             <family>Bahnschrift Light</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>---</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="lblTitle_3">
           <property name="font">
            <font>
             <family>Bahnschrift</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Last Name:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="lblLastName">
           <property name="font">
            <font>
             <family>Bahnschrift Light</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>---</string>
           </property>
          </widget>
         </item>
         <item row="10" column="0" colspan="2">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="6" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QRadioButton" name="rbFemaleConfirm">
             <property name="text">
              <string>Female</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="rbMaleConfirm">
             <property name="text">
              <string>Male</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="lblTitle_6">
           <property name="font">
            <font>
             <family>Bahnschrift</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Birthday:</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="lblDay">
             <property name="font">
              <font>
               <family>Bahnschrift Light</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>--</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lblSpace">
             <property name="font">
              <font>
               <family>Bahnschrift Light</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>/</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lblMouth">
             <property name="font">
              <font>
               <family>Bahnschrift Light</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>--</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lblSpace_2">
             <property name="font">
              <font>
               <family>Bahnschrift Light</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>/</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lblYear">
             <property name="font">
              <font>
               <family>Bahnschrift Light</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>--</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>585</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>cbDay</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>lblDay</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>355</y>
    </hint>
    <hint type="destinationlabel">
     <x>315</x>
     <y>376</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbMouth</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>lblMouth</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>159</x>
     <y>346</y>
    </hint>
    <hint type="destinationlabel">
     <x>357</x>
     <y>380</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbYear</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>lblYear</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>261</x>
     <y>343</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>378</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbMale</sender>
   <signal>clicked()</signal>
   <receiver>rbMaleConfirm</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>28</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>448</x>
     <y>293</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbFemale</sender>
   <signal>clicked()</signal>
   <receiver>rbFemaleConfirm</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>38</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel">
     <x>365</x>
     <y>304</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
